### ISSUE TITLE
[Accessibility]A11y_VSCode_ViewIntegratedTerminal_ScreenReader: NVDA does not announce any information when a new terminal gets added or deleted.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#52304]
**Environment Details:** 
VSCode Version : 1.24.0
OS Version : Win10

**Additional Details:**
MAS Violated: MAS1.3.1
ScreenReader Version: NVDA2018.2

**Repro Steps:**
1. Launch VS Code while NVDA is ON.
2. Go to View menu and Select Integrated Terminal.
3. Tab till Add Terminal button(+) or delete terminal and press enter

**Actual:**
NVDA does not announce any information when a new terminal gets added or deleted. When a new terminal opens, it also gets added to combobox and same happens when we delete any terminal.

**Expected:**
NVDA should announce some information when any new terminal gets added or deleted and also the current terminal where user is working.

**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

**User Impact:**
User will not get the proper information about new terminals.

**MAS Reference**
https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

Attachment for Reference: 
[A11y VSCode Terminal NVDA Add Delete Terminal.pptx](https://github.com/Microsoft/vscode/files/2114717/A11y.VSCode.Terminal.NVDA.Add.Delete.Terminal.pptx)

Does this issue occur when all extensions are disabled?: Yes

**Comment by isidorn** [comment]
This feel like the terminal service could simply do `aria.alert` whenever a new terminal is opened. Similar to how we do in debug lang. Feels like low hanging fruit, assigning to july

**Comment by alexr00** [comment]
It doesn't look like this is as simple as adding `aria.alert` whenever a terminal is opened or closed. The alert gets interrupted by whatever has focus next. On terminal start, this is the text in the terminal. On terminal stop this is whatever document is open or the mail canvas.

**Comment by Tyriar** [comment]
@AccessibilityTestingTeam-TCS what is the ideal behavior given the number of announcements that can follow? Would it be best to say "Terminal created, terminal focused, c colon backslash blah blah"?

**Comment by Tyriar** [comment]
The answer to this is that we don't need to announce added or deleted terminals but rather announce the focused terminal only. If the active terminal dies we don't need to announce it, just announce the other terminal or editor that's focused afterwards.

**Comment by Tyriar** [comment]
@alexr00 do you have enough info to do this for October?

**Comment by alexr00** [comment]
@Tyriar, the current behavior is that the screen reader starts reading the contents of the next item to be focused. Instead, if the terminal has focus and then dies we should announce what gained focus and not the contents of the thing that gained focus?

**Comment by alexr00** [comment]
I see. So the change here is to announce "Terminal <number>" every time the terminal gains focus. The cwd is already read after the terminal is focused, though not immediately.

**Comment by alexr00** [comment]
@Tyriar, what about when the terminal is split? Just announce the terminal number again?

**Comment by Tyriar** [comment]
@alexr00 yes, yes 😃
